### DISCUSSION TITLE
Dashy

### DISCUSSION THREAD

**Discussion by jschenke488** [catppuccin/catppuccin#2191] [Port Requests]
### Will you make the port?

I've got a lot of exams coming up, so it may take a while for me to make a port.

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://dashy.to/docs/theming
https://github.com/Lissy93/dashy/tree/master/src/styles

### Any additional comments?

Source: https://github.com/Lissy93/dashy

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by raycadle** [comment]
I would love for this to be adopted by the Catppuccin community. I use Catppuccin everywhere in my Hyprland setup (hypr, waybar, rofi, userstyles, etc) and would love to be able to theme my dashboard as well.

I thought about giving it a shot but I have no experience working with CSS (yet), let alone creating a Catppuccin port. Would love to help, given a bit of guidance.
